wasmtime/_func.py

```diff
@@ -141,7 +141,13 @@ class Caller(object):
     def __init__(self, ptr):
         self.__ptr__ = ptr
 
-    def get_export(self, name):
+    def __getitem__(self, name):
+        ret = self.get(name)
+        if ret is None:
+            raise IndexError("failed to find export {}".format(name))
```

Comment:
So, Python has a `LookupError` exception with two subclasses, `IndexError` and `KeyError`. `IndexError` is for, well, indexes (i.e. integers pointing into sequences), `KeyError` is for keys (any sorts of objects used to look up mappings).

This one should be `KeyError`.